diff --git a/dlls/winex11.drv/window.c b/dlls/winex11.drv/window.c
index ee08000ca78..73a616a5ebb 100644
--- a/dlls/winex11.drv/window.c
+++ b/dlls/winex11.drv/window.c
@@ -518,7 +518,7 @@ static unsigned long get_mwm_decorations_for_style( DWORD style, DWORD ex_style
 {
     unsigned long ret = 0;
 
-    if (X11DRV_HasWindowManager( "Mutter" )) return 0;
+    // if (X11DRV_HasWindowManager( "Mutter" )) return 0;
 
     if (ex_style & WS_EX_TOOLWINDOW) return 0;
     if ((ex_style & (WS_EX_LAYERED | WS_EX_COMPOSITED)) == WS_EX_LAYERED) return 0;
